client/src/components/todolist/TodoList.tsx

```diff
@@ -0,0 +1,62 @@
+import { graphql } from "babel-plugin-relay/macro";
+import { useFragment } from "react-relay";
+import { TodoList_tasks$key } from "./__generated__/TodoList_tasks.graphql";
+import "./Style.css";
+
+interface Props {
+  className: string;
+  tasks: TodoList_tasks$key;
+}
+
+const TodoList = (props: Props) => {
+  const data = useFragment(
+    graphql`
+      fragment TodoList_tasks on Query {
+        allTodoCreateTables {
+          nodes {
+            id
+            completed
+            task
+          }
```

Comment:
We can actually take this one step further and create another component for each individual node. In this fragment we can spread another fragment and create a new component that is only repsonsible for displaying each todo item. ie: TodoListItem.tsx
```suggestion
          edges {
           node {
              ...TodoListItem_todo
            }
          }
```